src/scroll-timeline-base.js

```diff
@@ -721,17 +721,14 @@ function parseInset(value) {
   let parts = value;
   // Parse string parts to
   if (typeof value === 'string') {
-    // Split value into separate parts
-    const stringParts = value.split(/(?<!\([^\)]*)\s(?![^\(]*\))/);
-    parts = stringParts.map(str => {
-      if (str.trim() === 'auto') {
+    parts = splitIntoComponentValues(value).map(str => {
+      if (str === 'auto') {
         return 'auto';
-      } else {
-        try {
-          return CSSNumericValue.parse(str);
-        } catch (e) {
-          throw TypeError('Invalid inset');
-        }
+      }
+      try {
+        return CSSNumericValue.parse(str);
+      } catch (e) {
+        throw TypeError('Invalid inset');
```

Comment:
For more clarity we should include the inset to give author a clue.

```suggestion
        throw TypeError(`Could not parse inset "${value}"`);
```